### PR TITLE
PLT-3644 - Updating link previews behaviour

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
#### Summary
Just toggling the link previews and also changing the default for display mode.
https://mattermost.atlassian.net/browse/PLT-3644

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [ ] Has driver changes that have been merged and package.json updated
- [ ] Has enterprise changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization files updated
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

File: webapp/components/user_settings/user_settings_display.jsx [hunk 1]
```diff
@@ -237,7 +237,7 @@ export default class UserSettingsDisplay extends React.Component {
                 title={
                     <FormattedMessage
                         id='user.settings.display.collapseDisplay'
-                        defaultMessage='Auto Collapse Previews'
+                        defaultMessage='Link previews'
```
Comment (enahum) [inline webapp/components/user_settings/user_settings_display.jsx]: @asaadmahmoodspin please edit this message in `webapp/i18n/en.json`

Comment (asaadmahmoodspin) [inline webapp/components/user_settings/user_settings_display.jsx]: @enahum The updated text is already there, it just wasn't updated in this file.

**Comment by enahum** [comment]
+1

**Comment by coreyhulen** [comment]
+1

**Pull request merged**